docs/sources/alerting/alerting-rules/view-alert-rules.md

```diff
@@ -36,6 +39,14 @@ To view alerting details:
 
 {{< figure src="/static/img/docs/alerting/unified/rule-details-8-0.png" max-width="650px" caption="Alerting rule details" >}}
 
+From the Alert list page, you can also make copies of alert rules to help you reuse existing alert rules.
+
+## Export alert rules
+
+Click **Export** to create and tune an alert rule in the UI, then export to YAML or JSON, and use in the provisioning API or files. You can also export an entire rule group to review or use.
```

Comment:
in the UI, we only export JSON, so far

Reply:
Update: we've just merged this [PR](https://github.com/grafana/grafana/pull/62760) for using yaml as default format.